libsolidity/codegen/ir/Common.cpp

```diff
@@ -68,3 +68,13 @@ string frontend::buildInternalDispatchFunctionName(Arity const& _arity)
 		"_in_" + to_string(_arity.in) +
 		"_out_" + to_string(_arity.out);
 }
+
+FunctionDefinition const* frontend::getReferencedFunctionDeclaration(Expression const& _expression)
```

Comment:
```suggestion
FunctionDefinition const* frontend::referencedFunctionDeclaration(Expression const& _expression)
```

Reply:
See my answer in https://github.com/ethereum/solidity/pull/8943#discussion_r425279907

Reply:
Done in #8952. It's hard to come up with a good name for the struct until there's more functions that create a pattern so I just named it `IRHelpers` for now.